app/routes/_index/NoticeFormatsAd.tsx

```diff
@@ -54,7 +54,7 @@ export function NoticeFormatsAd() {
         headerFirst
       >
         <CardHeader>
-          <p className="margin-0">Recommended</p>
+          <p className="margin-0">Recommended for classic topics</p>
```

Comment:
It's not really recommended, though, is it?

Reply:
It's our recommended way to receive the classic topics, as opposed to via socket, email, or VOEvent.

Reply:
I'm worried that this is a bit cryptic and self-referential.